ipsource.go

```diff
@@ -238,6 +241,70 @@ func (u UPnP) GetIPs(ctx context.Context, _ IPVersions) ([]net.IP, error) {
 	return []net.IP{ip}, nil
 }
 
+// CMD is an IP source that looks up the public IP addresses by
+// executing a script or command from your filesystem.
+//
+// The command must return the IP addresses comma spreaded in plain text.
+type CMD struct {
```

Comment:
I suggest naming it `Command` instead. "CMD" isn't an initialism.

Reply:
✅